scripts/reduce_notebook_runtime.py

```diff
@@ -13,11 +13,11 @@ def modify_notebook(file_path):
         "febrl3.ipynb"
     )
 
-    # if file_path.endswith("febrl4.ipynb"):
-    #     # These cells need a very high max_pairs value
-    #     # otherwise you get divide by zero errors.  Easiest just to delete
-    #     data["cells"] = data["cells"][:19]
-    #     changed = True
+    if file_path.endswith("febrl4.ipynb"):
```

Comment:
For some reason this has started failing again with the 'unable to take the logarithm of zero' error.

This is possibly just a random sampling thing, but in any case, it's not a problem.  It's only happening because on the testrunner I dramatically reduce the number of sampled pairs to make it run quickly, but this results in odd parameter estiamtes.  So better not to run this code at all